decidim-core/app/packs/src/decidim/sw/loader.js

```diff
@@ -0,0 +1,18 @@
+// check if the browser supports serviceWorker at all
+window.addEventListener("load", async () => {
+  if ("serviceWorker" in navigator) {
+    // eslint-disable-next-line no-unused-vars
+    const registration = await navigator.serviceWorker.register("/sw.js", { scope: "/" });
+
+    // NOTE: Uncomment this when enable PUSH notifications
+    // const permission = await window.Notification.requestPermission();
+
+    // if (permission !== "granted") {
+    //   throw new Error("Permission not granted for Notification");
+    // }
+
+    // do stuff
+  } else {
+    console.log("Your browser doesn't support service workers 🤷‍♀️");
+  }
```

Comment:
@Crashillo please remove the comments and fix the else

Reply:
Why so? Tests are green and that's gonna be modified in the push notifications PR, hence you only have to uncomment things

Reply:
So let's add it when that's necessary. I don't like adding commented code for future features, either.

Reply:
Can you remove these comments @Crashillo 🙏🏽 ?